AWSome.Chatter/Objects/MessageObject.cs

```diff
@@ -1,15 +1,22 @@
 ﻿using System.Xml.Serialization;
 
-namespace AWSome.Messages.Objects
+namespace Liberator.AWSome.Chatter.Objects
 {
     /// <summary>
     /// Abstract class to form the basis of Kinesis messages
     /// </summary>
     public abstract class MessageObject
     {
+        /// <summary>
+        /// 
+        /// </summary>
         [XmlIgnore]
         public string PartitionKey { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="PartitionKey"></param>
         protected MessageObject(string PartitionKey)
         {
             this.PartitionKey = PartitionKey;
```

Comment:
empty summaries in this class?

Reply:
How did that happen? Oh, yeah ... me ... d'oh!